### PR TITLE
TELCODOCS-294: Created new module for FIPS validation and moved additional resources into RHCOS module

### PR DESCRIPTION
**Pull request by MirzWeiss**
OCP 4.9 documentation for [KATA-579](https://issues.redhat.com/browse/KATA-579)

Documentation preview: https://deploy-preview-35937--osdocs.netlify.app/openshift-enterprise/latest/sandboxed_containers/understanding-sandboxed-containers.html#security-compliance-nist_understanding-sandboxed-containers

Approved by SMEs.

@zanetworker @snir911

@valtair1891 - can you review and move the doc user story to QA in progress? (https://issues.redhat.com/browse/TELCODOCS-294)

### CODE REVIEW

**Comment by MirzWeiss** [comment]
Documentation for [KATA-579](https://issues.redhat.com/browse/KATA-579)

Documentation preview: https://deploy-preview-35937--osdocs.netlify.app/openshift-enterprise/latest/sandboxed_containers/understanding-sandboxed-containers.html#security-compliance-nist_understanding-sandboxed-containers

@zanetworker @snir911

@valtair1891 - can you review and move the doc user story to QA in progress? (https://issues.redhat.com/browse/TELCODOCS-294)

**Comment by zanetworker** [comment]
This does not seem to be the first PR adding FIPS docs. Found a couple of nits in older commits: 

> FIPS compliance is one of the most critical components required in highly secure environments, to ensure that only supported cryptographic technologies are allowed on nodes.

Can we remove "most" and just use "is a critical component".  I am not in favor of exaggeration but that is just my personal preference. 

> [IMPORTANT]
====
The use of FIPS Validated / Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
====

This sentence "The use of FIPS Validated / Modules" looks wrong. Should it be "The use of FIPS Validated modules"?

**Comment by MirzWeiss** [comment]
> Can we remove "most" and just use "is a critical component". I am not in favor of exaggeration but that is just my personal preference.

This is an existing section that was already written about FIPS. Since I am not the original author, I can't change this without finding out who was, and checking with them and their SME whether we can change it.

> > [IMPORTANT]
> > ====
> > The use of FIPS Validated / Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
> > ====
> 
> This sentence "The use of FIPS Validated / Modules" looks wrong. Should it be "The use of FIPS Validated modules"?

It's "The use of FIPS Validated / Modules in Process" -FIPS Validated cryptographic libraries, and Modules in Process cryptographic libraries. Both are referenced here.

**Comment by zanetworker** [comment]
@MirzWeiss I did not get the use "/" still in the sentence. But maybe it is just me.

**Review by snir911 — APPROVED** [review]
LGTM, thanks!

**Comment by MirzWeiss** [comment]
> @MirzWeiss I did not get the use "/" still in the sentence. But maybe it is just me.

I'm going to look into this, but I believe it's out of scope of this issue. This is used multiple times as is throughout the documentation. I've filed a bug for it: https://bugzilla.redhat.com/show_bug.cgi?id=2003741

File: modules/sandboxed-containers-fips-validation.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+[id="sandboxed-containers-fips-validation_{context}"]
+
+= FIPS validation for {sandboxed-containers-first}
+
+The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. The sandboxed containers {op-system} extension contains RPMs for Kata, QEMU, and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.
```
Comment (sagidlow) [inline modules/sandboxed-containers-fips-validation.adoc:9]:
```suggestion
The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. The {sandboxed-containers-first} {op-system} extension contains RPMs for Kata, QEMU, and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.
```

**Review by sagidlow — COMMENTED** [review]
Just one nit comment. Otherwise LGTM.

File: modules/sandboxed-containers-fips-validation.adoc (hunk 1 above)
Comment (MirzWeiss) [reply modules/sandboxed-containers-fips-validation.adoc]: My bad - this was actually a draft module I started writing at first before I realized a similar module already existed and I used it instead... The content here is for RHCOS. Glad you caught it - I deleted it and committed the changes. :)

**Pull request merged**